### ISSUE TITLE
Brew formula

### ISSUE THREAD

**Issue by andreiborisov** [sandreas/m4b-tool#18]
It would be convenient to install utility system-wide with `brew`

**Comment by sandreas** [comment]
Thank you for pointing this out.

Yes, this would be very nice... unfortunately there are a few things missing. At first i would like to create a small "homepage" for the tool (brew does not accept packages without homepage).

At the moment i'm to busy to do anything but perhaps in the beginning of the next year...
As soon as the homepage is out, i'll create a brew formulae (this should be too complex).

As an alternative, i would like to provide a preconfigured docker image with all dependencies in it and environment variables to configure the input and output...
Dockerfile is already in the repository, but it can only merge things so far.

**Comment by sandreas** [comment]
Progress: http://m4b-tool.fynder.de/
I know that it is very basic, but at least it qualifies for a pull request :-)

In the next days i will take a look at the brew formula... if someone would like to help, i would appreciate it.

**Comment by sandreas** [comment]
Ok, found the docs, brew formula should be working as expected with best audio quality options and all needed tools and required options. I ready updated the documentation.

If something does not work, please reopen this issue.

**Comment by schrodincat** [comment]
Wow, awesome! Thank you🙏

**Comment by ragaskar** [comment]
Use with caution, trying to install from this tap broke my functioning ffmpeg on catalina.

**Comment by sandreas** [comment]
> trying to install from this tap broke my functioning ffmpeg on catalina

This is a serious issue, thanks for reporting - I'll take a look.

**Comment by ragaskar** [comment]
Yeah, no worries, a `brew uninstall ffmpeg` followed by a `brew install ffmpeg` fixed it. ffmpeg even with vanilla brew seems to be voodoo, I recall having to do a number of weird steps when I first installed on this machine to avoid weird library errors. If it's helpful, when doing a brew install m4b-tool I got `Error: ffmpeg is already installed from homebrew/core!
Please `brew uninstall ffmpeg` first."`

The errors I got after running that command (fresh terminal window, so env was reloaded) when running `ffmpeg` directly were

```
○ → ffmpeg
dyld: Library not loaded: /usr/local/opt/x264/lib/libx264.159.dylib
  Referenced from: /usr/local/bin/ffmpeg
  Reason: image not found
Abort trap: 6
```

I'm not fussed about having it fixed (like I said, I'll probably try out the docker image approach when I get a chance), but love that you're taking a look after this feedback, so cool.
